packages/cli/src/commands/integration/open-integration.ts

```diff
@@ -0,0 +1,58 @@
+import chalk from 'chalk';
+import open from 'open';
+import type Client from '../../util/client';
+import getScope from '../../util/get-scope';
+import type { Configuration } from './types';
+import { fetchMarketplaceIntegrations } from './client';
+
+export async function openIntegration(client: Client, args: string[]) {
+  if (args.length > 1) {
+    client.output.error('Cannot open more than one dashboard at a time');
+    return 1;
+  }
+
+  const integrationSlug = args[0];
+
+  if (!integrationSlug) {
+    client.output.error('You must pass an integration slug');
+    return 1;
+  }
+
+  const { team } = await getScope(client);
+
+  if (!team) {
+    client.output.error('Team not found');
+    return 1;
+  }
+
+  let configuration: Configuration | undefined;
+
+  try {
+    configuration = (
+      await fetchMarketplaceIntegrations(client, integrationSlug)
+    )[0];
+  } catch (error) {
+    client.output.error(
+      `Failed to fetch configuration for ${chalk.bold(`"${integrationSlug}"`)}: ${(error as Error).message}`
+    );
+    return 1;
+  }
```

Comment:
suggestion: wrap in a `getConfiguration` method

Reply:
I'll wrap this slightly differently, specifically because I don't want to mask the actual `getConfiguration` concept in the client (searching by ID). Technically I'm using a `getConfigurations` endpoint in the back with some filtering, though I've only ever seen it return an array of 1 so far.